fastembed/text/e5_onnx_embedding.py

```diff
@@ -25,6 +25,16 @@
             "hf": "xenova/paraphrase-multilingual-mpnet-base-v2",
         },
     },
+    {
+        "model": "intfloat/multilingual-e5-large-instruct",
+        "dim": 1024,
+        "description": "multilingual model, e5-large-instruct",
+        "size_in_GB": 1.03,
+        "sources": {
+            "url": "https://huggingface.co/yashvardhan7/fast-multilingual-e5-large-instruct/blob/main/fast-multilingual-e5-large-instruct%20.tar.gz",
+            "hf": "multilingual-e5-large-instruct-onnx",
```

Comment:
Please push to Huggingface Hub as a model and add as "hf" here. There are enough models added to demonstrate how this is done.

Reply:
In order to push it on Qdrant account on HuggingFaceHub , I need to join it. As on my account 'hf' is yashvardhan7/fast-multilingual-e5-large-instruct